trpcultivate_phenotypes/tests/src/Kernel/Validators/MetadataInputTest.php

```diff
@@ -408,6 +443,18 @@ public function testProjectGenusInput() {
       'Project genus match validator case title does not match expected title for a valid project+genus.');
     $this->assertTrue($validation_status['valid'], 'A valid project+genus must return a TRUE valid status.');
     $this->assertEmpty($validation_status['failedItems'], 'A valid project+genus does not return a failed item value.');
+
+    // Test a project with multiple genus.
+    $project = $this->test_project['project-with-2-configgenus']['id'];
+    $genus = $this->test_genus['configured-secondary'];
+
+    $form_values = ['project' => $project, 'genus' => $genus];
+    $validation_status = $instance->validateMetadata($form_values);
+
+    $this->assertEquals('Project exists and project-genus match the genus provided', $validation_status['case'],
+      'Project genus match validator case title does not match expected title for a valid project+genus (project with multiple genus).');
+    $this->assertTrue($validation_status['valid'], 'A valid project+genus must return a TRUE valid status.');
+    $this->assertEmpty($validation_status['failedItems'], 'A valid project+genus does not return a failed item value.');
```

Comment:
I wonder if this PR is a good opportunity to separate out `testProjectGenusInput()` from MetadataInputTest.php? Some code was added to `setUp()` in this PR which I assume is not needed for `testGenusInput()`. This PR could create a new test file specific for testing the ProjectGenusMatch validator and leave the pre-existing test and setUp() for GenusExists() in this file for a future PR to deal with.

Reply:
I agree with Carolyn on this and would like to see this moved into it's own file in this PR.